### PR TITLE
Add FuseML extension development tutorial for OpenVINO

### PR DESCRIPTION
**Pull request by stefannica**
Available for preview here: https://stefannica.github.io/docs/dev/tutorials/openvino-extensions/

### CODE REVIEW

File: docs/tutorials/openvino-extensions.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+            selector: control-plane=controller-manager
+            namespace: ovms-operator-system
+            condition: available
+            timeout: 600
+    uninstall:
+      - type: kustomize
+        location: kustomize
+    ```
+
+* adding a `services` section informs FuseML to automatically register the OVMS operator with the FuseML Extension Registry as a service that can be accessed from workflow steps. FuseML workflow steps such as the OVMS predictor, that need the OVMS operator to be preinstalled, will need to reference this extension by specifying an extension requiremenet with a matching `service_resource` field. The _internal_ endpoint URL indicates that the OVMS operator can only be used by workflow steps running in the same Kubernetes cluster (i.e. through a service account). As a side note, it is also be possible to create configurations to allow workflow steps like the OVMS predictor to deploy OVMS instances in remote Kubernetes clusters, provided that Kubernetes credentials are also supplied, but that is not covered here:
```
Comment (flaviodsr) [inline docs/tutorials/openvino-extensions.md:175]: typo `requiremenet` and there is a trailing space at the end of the line.

File: docs/tutorials/openvino-extensions.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+images:
+- name: controller
+  newName: quay.io/openvino/ovms-operator
+  newTag: 0.1.0
+```
+
+Developing a FuseML Installer Extension is done by assembling together a `description.yaml` file with installation instructions and other useful information relevant from an integration perspective. Following are sections from the `description.yaml` descriptor created for the OVMS operator:
+
+* the main section describes the product/version and points to the above Kustomize file for installing/uninstalling the OVMS operator:
```
Comment (flaviodsr) [inline docs/tutorials/openvino-extensions.md:151]: trailing space at the end of the line.

File: docs/tutorials/openvino-extensions.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+### FuseML OpenVINO Model Converter Step
+
+This workflow step leverages the OpenVINO Model Optimizer utilities and is required to convert models to the OpenVINO supported formats (IR) and model repository file structure.
+
+The container image required for this workflow step can be constructed using a prebuilt OpenVINO toolkit container image (~6GB in size, out of which 4GB are python packages for AI/ML) that provides all the necessary tools, or [building a custom image](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_docker_linux.html). For the sake of simplicity and to reduce technical debt, the former is used: the official OpenVINO toolkit container image is used as a base image and some additional software package requirements are installed on top.
+
+Aside from actual model conversion, the converter step also has to play a second function, that of interfacing with various object storage services (S3/AWS, GCS) used to store ML models. To be able to do that, it must be able to act as a client, and it does that by means of the minio client, which convenienly has support for all three: Minio, AWS S3 storage and Google Cloud Storage.
```
Comment (flaviodsr) [inline docs/tutorials/openvino-extensions.md:333]: typo `convenienly`

**Review by flaviodsr — CHANGES_REQUESTED** [review]
I think it looks great, however there are some new typos 😄

File: docs/tutorials/openvino-extensions.md (hunk 1 above)
Comment (stefannica) [reply docs/tutorials/openvino-extensions.md]: fixed

File: docs/tutorials/openvino-extensions.md (hunk 3 above)
Comment (stefannica) [reply docs/tutorials/openvino-extensions.md]: fixde

File: docs/tutorials/openvino-extensions.md (hunk 2 above)
Comment (stefannica) [reply docs/tutorials/openvino-extensions.md]: yup

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**